packages/agent-builder/src/defaults.ts

```diff
@@ -408,8 +407,8 @@ export const mastra = new Mastra({
     network: 'src/mastra/networks',
   };
 
-  static DEFAULT_TOOLS = async (projectPath: string, mode: 'template' | 'code-editor' = 'code-editor') => {
-    const agentBuilderTools = {
+  static DEFAULT_TOOLS = async (projectPath: string) => {
+    return {
```

Comment:
All these changes are just returning all the tools as an object, rather than having filtering logic inside here.